ui.tests/test-module/specs/recaptcha/recaptcha.runtime.spec.js

```diff
@@ -112,35 +113,129 @@ describe("Form Runtime with Recaptcha Input", () => {
         })
     }
 
+    function updateRecaptchaSecretKey(secretKey) {
+        cy.openPage("/mnt/overlay/fd/af/cloudservices/recaptcha/properties.html?item=%2Fconf%2Fcore-components-it%2Fsamples%2Frecaptcha%2Fbasic%2Fsettings%2Fcloudconfigs%2Frecaptcha%2Fv2checkbox").then(x => {
+            cy.get('#recaptcha-cloudconfiguration-secret-key').clear().type(secretKey);
+            cy.get("#shell-propertiespage-doneactivator").click();
+        })
+    }
+
+
+    it("client side validation should fail if recaptcha is not filled", () => {
+        cy.previewForm(v2checkboxPagePath).then((p) => {
+            formContainer = p;
+        });
+        expect(formContainer, "formcontainer is initialized").to.not.be.null;
+        cy.get(`.cmp-adaptiveform-button__widget`).click().then(x => {
+            cy.get('.cmp-adaptiveform-recaptcha__errormessage').should('exist').contains('Please fill in this field.');
+        });
+    })
+
+    it("submission should pass for mandatory recaptcha v2", () => {
+        const secretKey="6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe";
+        updateRecaptchaSecretKey(secretKey);
+        cy.previewForm(v2checkboxPagePath).then((p) => {
+            formContainer = p;
+        });
+        expect(formContainer, "formcontainer is initialized").to.not.be.null;
+        cy.intercept('POST', '/adobe/forms/af/submit/*').as('submitForm');
```

Comment:
This will not work for context path, improve regex. Also update other places also.

Reply:
Done